core/summary_api.php

```diff
@@ -466,82 +499,38 @@ function summary_print_by_developer() {
 				ORDER BY handler_id, status';
 	$t_result = db_query( $t_query );
 
-	$t_last_handler = -1;
+	$t_summaryusers = array();
+	$t_cache = array();
+	$t_bugs_total_count = 0;
 	$t_bugs_open = 0;
 	$t_bugs_resolved = 0;
 	$t_bugs_closed = 0;
-	$t_bugs_total = 0;
```

Comment:
The above 4 assignments are unnecessary, the variables' values are overwritten in the foreach loop